dataland-backend/src/main/kotlin/org/dataland/datalandbackend/validator/DataPointMinimumValue.kt

```diff
@@ -0,0 +1,51 @@
+package org.dataland.datalandbackend.validator
+
+import jakarta.validation.Constraint
+import jakarta.validation.ConstraintValidator
+import jakarta.validation.ConstraintValidatorContext
+import jakarta.validation.Payload
+import org.dataland.datalandbackend.interfaces.datapoints.BaseDataPoint
+import org.dataland.datalandbackendutils.exceptions.InvalidInputApiException
+import java.math.BigDecimal
+import kotlin.properties.Delegates
+import kotlin.reflect.KClass
+
+/**
+ * Annotation for the validation of an ExtendedDataPoint<*> holding a number
+ */
+@Target(AnnotationTarget.FIELD)
+@Constraint(validatedBy = [DataPointMinimumValidator::class])
+annotation class DataPointMinimumValue(
+    val minimumValue: Long = 0,
```

Comment:
Why is it set to this value?

Reply:
by default, all numbers are assumed to be positive. Therefore the default minimum is zero. But as above, the minimum is manually set to zero again by the dev-tools for readability, so at the moment this is never used